spec/presenters/hyrax/etd_presenter_spec.rb

```diff
@@ -8,18 +8,24 @@
   let(:request)       { instance_double('Rack::Request', host: 'example.com') }
 
   let(:attributes) do
-    { title:         ['Moomin Title'],
-      creator:       ['Tove Jansson'],
-      resource_type: ['letter from moominpapa'],
-      source:        ['Too-Ticky'],
-      rights_note:   ['For the exclusive viewing of Little My.',
-                      'Moomin: do not read this.'],
-      subject:       ['Moomintrolls', 'Snorks'] }
+    { title:            ['Moomin Title'],
+      creator:          ['Tove Jansson'],
+      license:       ['A license'],
```

Comment:
If you're making changes anyway...  could you fix the spacing for line 13?

Reply:
Can fix this in rebase.